### ISSUE TITLE
FXML Loading cannot find controller class

### ISSUE THREAD

**Issue by ItsTheSky** [tlf30/jme-jfx#1]
Hello 👋 
While loading an FXML file, for example, the following one:
```xml
<?xml version="1.0" encoding="UTF-8"?>

<?import javafx.scene.layout.GridPane?>
<?import javafx.scene.layout.HBox?>
<?import javafx.scene.image.ImageView?>
<?import javafx.scene.text.Text?>
<GridPane xmlns:fx="http://javafx.com/fxml" fx:controller="info.itsthesky.game.states.MainMenuState.MainMenuController">
	<HBox>
		<ImageView fx:id="icon"/>
		<Text text="Planetizer" style="-fx-font-size: 32px;"/>
	</HBox>
</GridPane>
```
I defined the controller in the file directly. However, I'm getting a `ClassNotFoundException` and the controller cannot be found (of course it exists, and is accessible, in addition it was IntelliJ's auto-complete).

The "work-around" for this is to change the controller directly in the code, aka, from:
```java
return FXMLLoader.load(...);
```

To:

```java
final Object controller = new MainMenuState.MainMenuController(); // It's a sub-class
final FXMLLoader loader = new FXMLLoader(...);
loader.setController(controller);
return loader.load();
```
I'm opening this issue to know if you were warned about that, if a fix was / will be made or if it was normal and we simply have to use programmatical controller definition.

Have a nice day and thanks for your answer ^.^

**Comment by tlf30** [comment]
That is correct. I have been unable to load the controllers without manually setting it on the FXML Loader. I have only used groovy scripts as controllers and figured it was something to do with the groovy script engine reflection, but it seems that you experienced the exact same thing I did.

At this point in time, I would say it is a safe assumption that the controller will need to be manually specified. If someone else knows a fix, please share.
